bundles/binding/org.openhab.binding.wr3223/README.md

```diff
@@ -0,0 +1,133 @@
+# WR3223 ventilation controller Binding
+
+The ventilation controller WR3223 is used for room ventilation systems, for example in houses produced by company “Schwörer Haus”. The manufacturer of the controller is Hermes Electronic. (http://www.hermes-electronic.de/)
+
+The WR3223 binding can read several measurements and the current state of the ventilation system. It can also adjust the fan speed, operation mode and the target temperature.
+
+## Prerequisites
+
+The WR3223 can be connected by a RS232 or USB adapter. The adapter can be orderd by Hermes Electronic. If you wont connect the WR3223 by your self, have a look at: https://community.openhab.org/t/wr3223-ventilation-controller-schworer-haus/
+
+To controll the WR3223, you have to disconnect the control panel (Bedienteil), otherwise the WR3223 ignore the command from the serial port. If you have any trouble, just let me know.
+
+## Binding Configuration
+
+This binding can be configured in the file `services/wr3223.cfg`.
+
+All parameters are optional, but you have to configure at least the connection via serial port or via TCP. 
+
+| Property       | Default | Required | Description |
+|----------------|---------|:--------:|-------------|
+| serialPort     |         |   Yes (if serial port is used)     | Serial port where the WR3223 is connected. E.g. /dev/ttyUSB0 |
+| host           |         |   Yes (if TCP is used)     | IP address of the TCP to serial gateway |
+| port           |         |   Yes (if TCP is used)     | port number of the TCP to serial gateway  |
+| refresh        | 15000   |   No     | Refresh inverval in milliseconds. The WR3223 need at least every 20 second a refresh! |
+| controllerAddr | 1       |   No     | The controller address is normaly 1.
+
+## Item Configuration
+
+The syntax of the binding configuration strings accepted is the following:
+
+```
+wr3223="<device-command>"
+```
+
+where `<device-command>` is replaced with the WR3223 command from the list below:
+
```

Comment:
The manual is in German and I tried to translate the German names into englisch for the commands. If you wish, I can translate the notes to English. The reason why I let the notes in German is, because it's a German product and I thing the users mostly speak German.